src/Symfony/Component/Form/CHANGELOG.md

```diff
@@ -41,6 +42,7 @@ CHANGELOG
    ```
  * added the `input_format` option to `DateType`, `DateTimeType`, and `TimeType` to specify the input format when setting
    the `input` option to `string`
+ * added a `symbol` option to the `PercentType` that allows to disable or customise the output of the percent character   
```

Comment:
There are two entries (one here and on at the top), we should remove one :)

Reply:
Will do tomorrow 👍🏻